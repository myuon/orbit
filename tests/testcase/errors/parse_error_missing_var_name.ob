fun main() do
  let = 5;
end