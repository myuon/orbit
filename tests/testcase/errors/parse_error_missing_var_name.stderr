Expected variable name after 'let'